packages/sdk/src/types/ClientSide.ts

```diff
@@ -9,9 +9,12 @@ export interface ClientSideValidateParams {
 	code?: string
```

Comment:
Did you rebase your branch on top of master? I don't get why I see the same changes here which already have been applied in previous commit: https://github.com/voucherifyio/voucherify-js-sdk/commit/bf4e2ffdf018c4e6de9d256ea4a25e9b2d280cc6